flame/data.py

```
@@ -122,14 +131,14 @@ def randint(
         low: int,
         high: int,
         buffer_size: int = 1024,
-        g: torch.Generator = torch.Generator()
+        g: torch.Generator = torch.Generator(),
```

Ruff (0.8.2)

Do not perform function call `torch.Generator` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)